docs/source/gcclassic-user-guide/dry-run-run.rst

```diff
@@ -32,9 +23,27 @@ files <cfg>`:
 
    #. :option:`start_date`: Set the start date and time for your simulation.
    #. :option:`end_date`: Set the end date and time for your simulation.
-   #. :option:`met_field`: Check if the meteorology setting 
+   #. :option:`met_field`: Check if the meteorology setting
       (:option:`GEOS-FP`, :option:`MERRA2`, :option:`GCAP2`) is
       correct for your simulation.
+
+      .. important::
+
+         The convection scheme used for GEOS-FP met generation changed
+         from RAS to Grell-Freitas with impact on GEOS-FP meteorology
+         files starting June 1, 2020, specifically enhanced vertical
+         transport. In addition, there is a bug in convective
+         precipitation flux following the switch where all values are
+         zero. While this bug is automatically fixed by computing fluxes
+         online for runs starting on or after June 1 2020, the fix
+         assumes meteorology year corresponds to simulation year. Due to
+         these issues we recommend splitting up GEOS-FP runs in time such
+         that a single simulation does not run across June
+         1, 2020. Instead. set one run to stop on June 1 2020 and then
+         restart a new run from there. If you wish to use a GEOS-FP
+         meteorology year different from your simulation year please
+         create a GEOS-Chem GitHub issue for assistance.
```

Comment:
Since this copy of the GEOS-FP blurb is in the context of dry run maybe it should be more focused on the choice of met to download. "The convection scheme used for GEOS-FP met generation changed from RAS to Grell-Freitas with impact on GEOS-FP meteorology files starting June 1, 2020. For this reason we recommend using MERRA2 instead of GEOS-FP if running a simulation across June 1, 2020 to avoid unexpected discontinuities. Additional information about the impact of the convection change is at https://github.com/geoschem/geos-chem/issues/1409."

Reply:
Now fixed in 09a5db3